src/main/scala/uk/gov/nationalarchives/tdr/api/service/DisplayPropertiesService.scala

```diff
@@ -0,0 +1,39 @@
+package uk.gov.nationalarchives.tdr.api.service
+
+import uk.gov.nationalarchives.Tables
+import uk.gov.nationalarchives.tdr.api.db.repository.DisplayPropertiesRepository
+import uk.gov.nationalarchives.tdr.api.graphql.fields.DisplayPropertiesFields._
+import uk.gov.nationalarchives.tdr.api.graphql.fields.DisplayPropertiesFields
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class DisplayPropertiesService(displayPropertiesRepository: DisplayPropertiesRepository)(implicit val ec: ExecutionContext) {
+
+  implicit class DisplayPropertyRowHelper(properties: Seq[Tables.DisplaypropertiesRow]) {
+    def toDisplayProperty: Seq[DisplayPropertyField] = {
```

Comment:
Does this need to be an implicit as it's only being used once? Couldn't it be a private method that display properties gets passed into?

Reply:
Fair point, getting a bit carried away with implicits